iothub/service/src/Jobs/Models/ScheduledJob.cs

```diff
@@ -28,72 +53,55 @@ public class ScheduledJob
-        public DateTime? CreatedTimeUtc { get; internal set; }
+        public DateTime? CreatedOnUtc { get; internal set; }
 
         // Some service Jobs APIs use "createdTime" as the key for this value and some others use "createdDateTimeUtc".
         // This private field is a workaround that allows us to deserialize either "createdTime" or "createdDateTimeUtc"
         // as the created time value for this class and expose it either way as CreatedTimeUtc.
         [JsonProperty(PropertyName = "createdTime", NullValueHandling = NullValueHandling.Ignore)]
-        internal DateTime? _alternateCreatedTimeUtc
+        internal DateTime? AlternateCreatedOnUtc
         {
-            get => CreatedTimeUtc;
-            set => CreatedTimeUtc = value;
+            get => CreatedOnUtc;
+            set => CreatedOnUtc = value;
         }
 
         /// <summary>
         /// System generated. Ignored at creation.
         /// </summary>
         [JsonProperty(PropertyName = "startTimeUtc", NullValueHandling = NullValueHandling.Ignore)]
-        public DateTime? StartTimeUtc { get; internal set; }
+        public DateTime? StartedOnUtc { get; internal set; }
 
         /// <summary>
         /// System generated. Ignored at creation.
         /// Represents the time the job stopped processing.
         /// </summary>
         [JsonProperty(PropertyName = "endTimeUtc", NullValueHandling = NullValueHandling.Ignore)]
-        public DateTime? EndTimeUtc { get; internal set; }
+        public DateTime? EndedOnUtc { get; internal set; }
 
         // Some service Jobs APIs use "endTime" as the key for this value and some others use "endTimeUtc".
         // This private field is a workaround that allows us to deserialize either "endTime" or "endTimeUtc"
         // as the created time value for this class and expose it either way as EndTimeUtc.
         [JsonProperty(PropertyName = "endTime")]
-        internal DateTime? _alternateEndTimeUtc
+        internal DateTime? AlternateEndedOnUtc
         {
-            get => EndTimeUtc;
-            set => EndTimeUtc = value;
+            get => EndedOnUtc;
+            set => EndedOnUtc = value;
         }
 
         /// <summary>
-        /// Max execution time in secounds.
-        /// </summary>
-        [JsonProperty(PropertyName = "maxExecutionTimeInSeconds", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
-        public long MaxExecutionTimeInSeconds { get; internal set; }
-
-        /// <summary>
-        /// [Required] The type of job to execute.
+        /// Max execution time.
         /// </summary>
-        [JsonProperty(PropertyName = "jobType", NullValueHandling = NullValueHandling.Ignore)]
-        [JsonConverter(typeof(StringEnumConverter))]
-        public JobType JobType { get; internal set; }
+        /// <remarks>The precision on this is in seconds.</remarks>
+        [JsonIgnore]
+        public TimeSpan MaxExecutionTime { get; set; }
```

Comment:
Present properties with TimeSpan, instead of seconds. Serializable property is now internal (just below).